db/database.go

```diff
@@ -25,9 +28,11 @@ func NewDatabase(cfg *chaindb.Config) (*Database, error) {
 	}
 
 	offerTable := chaindb.NewTable(db, offerPrefix)
+	swapTable := chaindb.NewTable(db, swapPrefix)
```

Comment:
The `Close()` function further down didn't change, so github won't let me comment there, but I don't see `swapTable` getting closed in it. You might move these 2 lines into the `Database` declaration, since the intermediate variables don't look like they are improving code readability.